tests/jwa_test.py

```diff
@@ -167,30 +175,33 @@ def test_verify_new_api(self) -> None:
         from cryptography.hazmat.primitives.asymmetric.ec import SECP256R1
 
         from josepy.jwa import ES256
+
         key = mock.MagicMock(key_size=256, curve=SECP256R1())
-        ES256.verify(key, b"message", b'\x00' * math.ceil(key.key_size / 8) * 2)
+        ES256.verify(key, b"message", b"\x00" * math.ceil(key.key_size / 8) * 2)
         assert key.verify.called is True
 
     def test_signature_size(self) -> None:
         from josepy.jwa import ES512
         from josepy.jwk import JWK
+
         key = JWK.from_json(
             {
-                'd': 'Af9KP6DqLRbtit6NS_LRIaCP_-NdC5l5R2ugbILdfpv6dS9R4wUPNxiGw'
-                     '-vVWumA56Yo1oBnEm8ZdR4W-u1lPHw5',
-                'x': 'AD4i4STyJ07iZJkHkpKEOuICpn6IHknzwAlrf-1w1a5dqOsRe30EECSN4vFxae'
-                     'AmtdBSCKBwCq7h1q4bPgMrMUvF',
-                'y': 'AHAlXxrabjcx_yBxGObnm_DkEQMJK1E69OHY3x3VxF5VXoKc93CG4GLoaPvphZQv'
-                     'Znt5EfExQoPktwOMIVhBHaFR',
-                'crv': 'P-521',
-                'kty': 'EC'
-            })
+                "d": "Af9KP6DqLRbtit6NS_LRIaCP_-NdC5l5R2ugbILdfpv6dS9R4wUPNxiGw"
```

Comment:
ok this one I disagree with.

Reply:
ok reading black's documentation, I bet it would look nice again if each value was enclosed in parens.

Reply:
How does that look for you?

I can also rebase and squash commits if you want since we don't want to let GitHub squash this, but I'm personally not sure it's worth worrying about.

Reply:
Much better, delighted to see that this worked as expected! Nah no need to rebase.